cmd/cosign/cli/options/sign.go

```diff
@@ -97,6 +98,10 @@ func (o *SignOptions) AddFlags(cmd *cobra.Command) {
 		"write the certificate to FILE")
 	_ = cmd.MarkFlagFilename("output-certificate", certificateExts...)
 
+	cmd.Flags().StringVar(&o.OutputBundle, "output-bundle", "",
+		"write the bundle to FILE")
```

Comment:
Minor: in `sign-blob` and `attest-blob` we just call this `--bundle`. Arguably `--output-bundle` is more descriptive, but `--bundle` would be more consistent. I slightly lean towards consistency here, but open to other opinions.

Separately, taking inspiration from `sign-blob` maybe the description could be something like:

```
write everything required to verify the image to a FILE
```